resources/js/components/Room/SettingsComponent.vue

```diff
@@ -1,14 +1,26 @@
 <template>
   <div>
     <b-form  @submit.stop.prevent :novalidate="true">
-      <b-overlay :show="saving" >
+      <b-overlay :show="saving || loadingRoomTypes" >
       <div class="row" v-on:keyup.enter="save" @change="save">
         <!-- General settings tab -->
         <div class="col-lg-3 col-sm-12">
           <h5>{{ $t('rooms.settings.general.title') }}</h5>
           <b-form-group :state="fieldState('roomType')" :label="$t('rooms.settings.general.type')">
             <b-input-group>
-            <b-form-select :state="fieldState('roomType')" v-model.number="settings.roomType" :options="roomTypeSelect"></b-form-select>
+              <b-form-select :state="fieldState('roomType')" v-model.number="settings.roomType" :options="roomTypeSelect">
```

Comment:
Since the construct is used twice, I would prefer to create a component for this and use the component here and in the NewRoomComponent!